homeassistant/components/automation/device.py

```diff
@@ -1,18 +1,28 @@
 """Offer device oriented automation."""
 import voluptuous as vol
 
-from homeassistant.const import CONF_DOMAIN, CONF_PLATFORM
+import homeassistant.components.device_automation as device_automation
+from homeassistant.const import CONF_DEVICE_ID, CONF_DOMAIN, CONF_PLATFORM
 from homeassistant.loader import async_get_integration
 
 
 # mypy: allow-untyped-defs, no-check-untyped-defs
 
 TRIGGER_SCHEMA = vol.Schema(
```

Comment:
We should expand the TRIGGER_BASE_SCHEMA from `device_automation` and add `extra=vol.ALLOW_EXTRA`

Reply:
```suggestion
TRIGGER_SCHEMA = device_automation.TRIGGER_BASE_SCHEMA.extend(
```

Reply:
Fixed